Testing new civet recipes

milljm opened the pull request:
Test if we can detect a failure to update packages after modifying an influential file being tracked by versioner.py.

Ref: #000

DO NOT MERGE

Commit:
Testing new civet recipes

Test if we can detect a failure to update packages after modifying
an influential file being tracked by versioner.py.

Ref: #000

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2487303/), step Conda build config check on 7c22814 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/mpi

The following file(s) are unchanged:

  conda/mpi/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2487303/), step Conda build config check on 7c22814 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/mpi

The following file(s) are unchanged:

  conda/mpi/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

Commit:
doing as instructed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2488079/), step Conda build config check on 20a75ad wanted to post the following:

A change of the following file(s) triggered this check:

  conda/mpi

The following file(s) are unchanged:

  conda/mpi/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

Commit:
testing

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2489498/), step Conda build config check on a44b539 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/peacock

The following file(s) are unchanged:

  conda/mpi/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

Commit:
adding tracking libraries to versioner

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2489663/), step Conda build config check on 3b74d13 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/peacock

The following file(s) are unchanged:

  conda/mpi/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

Commit:
Return specific error code

Return a specific error code when asking for hash for a library
that is not tracked in supplied hash||branch.

moosebuild commented:
All [jobs](https://civet.inl.gov/event/200915/) on ca9bdac : canceled by @milljm

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2490722/), step Conda build config check on ca9bdac wanted to post the following:

A change of the following file(s) triggered this check:

  conda/peacock

The following file(s) are unchanged:

  conda/mpi/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

moosebuild commented:
Job [Precheck](https://civet.inl.gov/job/2494755/), step Conda build config check on c43f1a5 wanted to post the following:

A change of the following file(s) triggered this check:

  conda/peacock

The following file(s) are unchanged:

  conda/mpi/conda_build_config.yaml

The conda build config was not changed but one or more of its dependencies have changed

milljm commented:
This PR has served its purpose. Closing.
